CMakeLists.txt

```diff
@@ -185,7 +185,7 @@ if (${RUN_TESTS} MATCHES "true")
 
         # Push and execute tests as post-build event
 
-        set(TARGET_TEST_DIR /data/local/tmp/mwengine_unittest/${ANDROID_ABI}) # Directory on device to push tests.
+        set(TARGET_TEST_DIR /data/local/tmp/mwengine_unittest_${ANDROID_ABI}) # Directory on device to push tests.
 
```

Comment:
Addresses issue where test devices on Android 6 could not generate nested folder.